hw/BUILD

```diff
@@ -21,6 +21,15 @@ string_list_flag(
     ],
 )
 
+# This configuration exposes fusesoc's "make_options" to enable parallel
+# compilation of the verilated model. Compilation takes about 30m of cpu time
+string_list_flag(
+    name = "make_options",
+    build_setting_default = [
+        "-j",
```

Comment:
I think this won't work. At least my machine (8 core, 16 threads, 32 GiB of RAM + 32 GiB of Swap) runs out of memory after 5 minutes. But at least this means the option has an effect. :-)

We have to choose a value that works reasonable on most developer machines. On my machine:
- `-j4` takes ~16 min, ~6 GiB RAM
- `-j8` takes ~13 min, ~11.3 GiB RAM

I would probably take `-j4` as the difference to `-j8` isn't huge but some people might still be using 4 core, 16 GiB laptops. I think in the past we also used `-j4`.

Reply:
I never worked out why it changed, so I only know that the present behavior on my machine and yours is weird, so I wanted to give us tools to control it and set a sane default. I'm pleased with -j 4 as a default, but if it was ever intentionally set, I'm not sure where, and it may have been unset because of confusion regarding various placements of -j options.

Reply:
With "in the past" I meant before the switch to bazel and in the core file itself (https://github.com/lowRISC/opentitan/blob/master/hw/top_earlgrey/dv/verilator/chip_sim.core) but it seems I misremembered things. We only set the number of threads to run the binary, not to build it.

Anyway, thanks for working on this @drewmacrae , I appreciate it!